drinks_touch/tasks/send_mail.py

```diff
@@ -90,25 +92,12 @@ def send_low_balance(self, account: Account):
 
     @with_db
     def send_summaries(self):
-        from sqlalchemy import inspect
-
         query = select(Account).where(Account.email.isnot(None) & Account.enabled)
         accounts = Session().scalars(query).all()
         for i, account in enumerate(accounts):
             self.progress = 0.5 + (i + 1) / len(accounts) / 2
             if self.sig_killed:
-                break
-            self.send_summary(account, "Getränkeübersicht")
```

Comment:
`send_summaries()` does nothing. Keeping `self.send_summary(account, "Getränkeübersicht")` should be fine

Reply:
The errors on this PR are skyrocketing… thanks for the thorough review… :sweat: